src/toGamut.js

```diff
@@ -24,7 +50,16 @@ import getColor from "./getColor.js";
  * @param {ColorSpace|string} options.space - The space whose gamut we want to map to
  */
 
-export default function toGamut (color, { method = defaults.gamut_mapping, space = color.space } = {}) {
+export default function toGamut (
+	color,
+	{
+		method = defaults.gamut_mapping,
+		space = color.space,
+		deltaEMethod = "",
+		jnd = 2,
+		blackWhiteClamp = {}
```

Comment:
Could you update the JSDoc comments above to reflect these new options?

Reply:
I can update comments. I'm not that familiar with type stuff in JS, so I may need very straight forward guidance for anything non-trivial, but I'll do what I can.

Reply:
Sure, I'm happy to help with type defs -- I think you both of these will be relatively straightforward changes, but let me know if you have questions.

`deltaEHCT` can duplicate the other `deltaE*` types in `types/src/deltaE/`.
`toGamut` needs the new options added in `types/src/toGamut.d.ts`, e.g.:

```ts
declare function toGamut (
	color: ColorTypes,
	options?: {
		method?: string | undefined;
		space?: string | ColorSpace | undefined;
		deltaEMethod?: string | undefined;
		jnd?: number | undefined;
		blackWhiteClamp?: {
			channel: Ref;
			min: number;
			max: number;
		} | undefined;
	} | string
): PlainColorObject;
```